src/mixer/playermanager.cpp

```diff
@@ -715,6 +715,21 @@ void PlayerManager::slotLoadTrackToPlayer(
             // so clone another playing deck instead of loading the selected track
             clone = true;
         }
+
+#ifdef __STEM__
+        // Reset the QuickFx of stem to their default value
+        if (m_pConfig->getValue(
+                    ConfigKey("[Mixer Profile]", "stem_auto_reset"), true)) {
+            ChannelHandleAndGroup handleGroup =
+                    m_pEngine->getChannelGroup(groupForDeck(m_decks.count()));
+            // Setup stem QuickEffect chain for this deck
+            for (int i = 0; i < 4; i++) {
```

Comment:
```suggestion
            for (int stemIdx  = 0; stemIdx  < mixxx::kMaxSupportedStems; stemIdx ++) {
```
Same below in line 810

Reply:
Fixed now